ui/src/app/pages/runtimes-list.tsx

```diff
@@ -10,6 +10,23 @@ import { NavigationProps } from 'app/utils/navigation';
 import { withNavigation } from 'app/utils/with-navigation-hoc';
 import { ajaxContext, Environments } from 'terraui/out/Environments';
 
+// Hidden columns Index
+const workspace = 2;
+const delete_cloud_environment = 11;
+const status_pd = 5;
+const delete_pd = 10;
+
+const hiddenTableColumn = [
```

Comment:
Use plural for lists unless there is a reason to deviate.
```suggestion
const hiddenTableColumns = [
```